app/controllers/doorkeeper/openid_connect/discovery_controller.rb

```diff
@@ -30,6 +30,7 @@ def provider_response
           introspection_endpoint: oauth_introspect_url(protocol: protocol),
           userinfo_endpoint: oauth_userinfo_url(protocol: protocol),
           jwks_uri: oauth_discovery_keys_url(protocol: protocol),
+          end_session_endpoint: Doorkeeper::OpenidConnect.configuration.end_session_endpoint.call(),
```

Comment:
Is this part exercised by a test? Could you point me the file, please? Thanks.

Reply:
I added a spec that this is added to the configuration here:
https://github.com/doorkeeper-gem/doorkeeper-openid_connect/pull/103/files#diff-db9ee42799b71b9a732daef09d2ebdcf

And another one for the controller to make sure it returns the value in the JSON response:
https://github.com/doorkeeper-gem/doorkeeper-openid_connect/pull/103/files#diff-d828e386ea3f9924e9fda0d952d9e561

Also, after you asking, I also added another spec to make sure that no end_session_endpoint is present in the discovery response if none was configured.

Am I missing anything?